test-projects/integration/charts.visivo.yml

```diff
@@ -0,0 +1,44 @@
+selectors:
+  - name: Common Selector
+    default: Simple Line
+    is_multi: false
+
+charts:
+  - name: Command Chart
+    # The on_ commands can take a chart, table, or selector reference
+    # It would then set the value of the selector to the selected trace/cohort from this item
+    on_select: ref(Common Selector)
```

Comment:
What's the difference between `on_select` here and the `selector` block on line 32?

Reply:
Couldn't we call both `selector`?

Reply:
Would it make more sense if the "on_select" was moved:

```
chart: 
  name: Chart
  selector:
    on_select: ref(Other Chart)
```

It might even make sense to call that line: `update_on_select` ?